tests/tree/test_tree.py

```diff
@@ -111,6 +111,18 @@ def test_faceDiv(self):
 
         self.assertEqual((M.faceDiv - T.permuteCC*T.faceDiv*T.permuteF.T).nnz, 0)
 
+    def test_serialization(self):
+        hx, hy = np.r_[1., 2, 3, 4], np.r_[5., 6, 7, 8]
+        mesh1 = discretize.TreeMesh([hx, hy], levels=2, x0=np.r_[-1, -1])
```

Comment:
it might actually be useful to do something with the TreeMesh, i.e. `mesh1.refine(2)`